cmd/goal/messages.go

```diff
@@ -69,6 +69,8 @@ const (
 	infoNodeCatchpointCatchupAccounts       = "Catchpoint total accounts: %d\nCatchpoint accounts processed: %d\nCatchpoint accounts verified: %d\nCatchpoint total KVs: %d\nCatchpoint KVs processed: %d\nCatchpoint KVs verified: %d"
 	infoNodeCatchpointCatchupBlocks         = "Catchpoint total blocks: %d\nCatchpoint downloaded blocks: %d"
 	nodeLastCatchpoint                      = "Last Catchpoint: %s"
+	nodeConfirmImplicitCatchpoint           = "Fast catchup to %s is about to start.\nUsing untrusted catchpoints or peers might compromise your node.\nType 'yes' to accept the risk and continue: "
```

Comment:
This seems like a very reasonable warning. Perhaps someone from product might want weigh in on the language. Maybe a harsher warning around participation? And maybe use language that doesn't say the node will be "compromised" (which maybe somebody things means vulnerable in the sense of "bad guys can infiltrate it").  Something about integrity of data being vulnerable, not the node itself.

I guess we might be worried that this breaks an automated catchup script.  Could we only fail if the node has participating accounts?

Reply:
> I guess we might be worried that this breaks an automated catchup script. Could we only fail if the node has participating accounts?

One could provision a node with fast catchup and add part keys after

Reply:
@algoanne do you want to weigh in here?

Reply:
I support us proceeding with this breaking change. 

Yes it might break automated scripts, but what are those automated scripts doing?
- relays require archival nodes so they don't use this command. 
- indexers require archival nodes so they don't use this command. 
- participating nodes shouldn't be using this command anyway, so we are glad to break their automated process and make them re-write it and re-consider what they're doing. 
- people using nodes just to submit transactions, I don't know how many there are - isn't a core use case I don't think. 

Do we need to update the command description as well as this output? 

As for wording, my suggestion (but I don't feel strongly)... 
`Fast catchup to %s is about to start.\nUsing external catchpoints is not a secure practice and should not be done for consensus participating nodes.\nType 'yes' to accept the risk and continue: `

Reply:
Applied Anne's wording.